data/json/items/comestibles.json

```diff
@@ -12210,6 +12210,36 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "frenchtoast",
+        "name" : "french toast",
```

Comment:
Shouln't it be "French toast"? Country names and derivatives should be capitalized.